streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -106,11 +119,12 @@
      *                 if not specified the default serde defined in the configs will be used
      * @param valSerde value serde used to send key-value pairs,
      *                 if not specified the default serde defined in the configs will be used
+     * @param streamName the name used to label the key/value pairs printed out to the console
      *
-     *                 Implementors will need to override toString for keys and values that are not of
-     *                 type String, Integer etc to get meaningful information.
+     * Implementors will need to override toString for keys and values that are not of
+     * type String, Integer etc to get meaningful information.
      */
-    void print(Serde<K> keySerde, Serde<V> valSerde);
+    void print(Serde<K> keySerde, Serde<V> valSerde, String streamName);
```

Comment:
We should have 4 overloaded `print` methods, ie, this one should not be replaced but kept.